description: fix string format warnings, output buffers were not large enough
author: Michael Gilbert <mgilbert@debian.org>

--- a/dlls/oleaut32/tmarshal.c
+++ b/dlls/oleaut32/tmarshal.c
@@ -333,7 +333,7 @@ static HRESULT reg_get_typelib_module(RE
     HKEY	ikey;
     REGSAM	opposite = (sizeof(void*) == 8) ? KEY_WOW64_32KEY : KEY_WOW64_64KEY;
     BOOL	is_wow64;
-    char	tlguid[200],typelibkey[300],interfacekey[300],ver[100];
+    char	tlguid[200],typelibkey[316],interfacekey[300],ver[100];
     char	tlfn[260];
     DWORD	tlguidlen, verlen, type;
     LONG	tlfnlen, err;
--- a/programs/winhlp32/hlpfile.h
+++ b/programs/winhlp32/hlpfile.h
@@ -26,7 +26,7 @@ typedef struct
 {
     char        type[10];
     char        name[9];
-    char        caption[51];
+    char        caption[132];
     POINT       origin;
     SIZE        size;
     int         style;
--- a/dlls/winealsa.drv/mmdevdrv.c
+++ b/dlls/winealsa.drv/mmdevdrv.c
@@ -630,7 +630,7 @@ HRESULT WINAPI AUDDRV_GetEndpointIDs(EDa
 static snd_config_t *make_handle_underrun_config(const char *name)
 {
     snd_config_t *lconf, *dev_node, *hu_node, *type_node;
-    char dev_node_name[64];
+    char dev_node_name[260];
     const char *type_str;
     int err;
 
--- a/dlls/wined3d/arb_program_shader.c
+++ b/dlls/wined3d/arb_program_shader.c
@@ -226,7 +226,7 @@ struct recorded_instruction
 
 struct shader_arb_ctx_priv
 {
-    char addr_reg[20];
+    char addr_reg[50];
     enum
     {
         /* plain GL_ARB_vertex_program or GL_ARB_fragment_program */
